### PR TITLE
create Lmcraft

### PR DESCRIPTION
**Pull request by Luispupuis**
## Type of change

- [ x] Add new directory for a server
- [ ] Update directory of a server
- [ ] Remove directory of a server
- [ ] Documentation Update

## Checklist

- [ x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x ] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [ x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/lmcraft/manifest.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "lmcraft",
+  "nice_name": "Lmcraft",
+  "direct_ip": "Lmcraft.de",
+  "server_wildcards": [
+    "%.Lmcraft.de"
+  ],
+  "social": {
+    "web": "https://Lmcraft.de",
```
Comment (maaaathis) [inline minecraft_servers/lmcraft/manifest.json:9]: Website not reachable

File: minecraft_servers/lmcraft/manifest.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "web_shop": "https://shop.Lmcraft.de",
```
Comment (maaaathis) [inline minecraft_servers/lmcraft/manifest.json:10]: Not reachable

File: minecraft_servers/lmcraft/manifest.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
+{
+  "server_name": "lmcraft",
+  "nice_name": "Lmcraft",
+  "direct_ip": "Lmcraft.de",
+  "server_wildcards": [
+    "%.Lmcraft.de"
```
Comment (maaaathis) [inline minecraft_servers/lmcraft/manifest.json:6]: Doesn't look like a real wildcard
image

**Review by maaaathis — CHANGES_REQUESTED** [review]

File: minecraft_servers/lmcraft/manifest.json (hunk 2 above)
Comment (Luispupuis) [reply minecraft_servers/lmcraft/manifest.json:10]: Comming soon. Server is not puplic yet

Comment (maaaathis) [reply minecraft_servers/lmcraft/manifest.json:10]: Then close this PR and reopen once your server is released. Only publicly available server can be added to the list.

**Comment by maaaathis** [comment]
_Server is not public yet._
Close this PR and reopen once your server is released. Only publicly available server can be added to the list.

**Pull request closed without merging**